axes/backends.py

```diff
@@ -0,0 +1,42 @@
+from django.contrib.auth.backends import ModelBackend
+from django.core.exceptions import PermissionDenied
+
+from axes.attempts import is_already_locked
+from axes.utils import get_lockout_message
+
+
+class AxesModelBackend(ModelBackend):
+
+    class RequestParameterRequired(Exception):
```

Comment:
This should be moved into `axes.exceptions` file (you can just create that) as per good Django package conventions. Exceptions are easier to find when they are in standard locations.

Reply:
I strongly disagree eith moving the exception and using suspicious operation, although im always open to naming changes. 

Moving indicates it is a failure on the module level, which i dont think applies here. The convention, to me, suggests a "Misconfigured" or "BadParameter" parent exc in exceptions.py that can be subclassed here, but i think the subclass should live here as it is tied to the backend not the module. 

Is this a suspicious operation? I think its more programmer error?

Reply:
But maybe im thinking app as part of a big project wise and not standalone library wise

Reply:
OK, I think it's valid to have this here as well. Would be nice to have some tests for this, but I think we can implement those later.